tripy/tests/spec_verification/test_dtype_constraints.py

```diff
@@ -181,3 +181,21 @@ def test_neg_dtype_constraints(test_data):
         api_call_locals, namespace = _run_dtype_constraints_subtest(test_data)
         if isinstance(api_call_locals[RETURN_VALUE], tp.Tensor):
             assert api_call_locals[RETURN_VALUE].dtype == namespace[return_dtype]
+
+
+operations = [obj.__qualname__ for _, obj in inspect.getmembers(tp) if inspect.isfunction(obj)]
```

Comment:
Could you use `tests.helper.get_all_tripy_interfaces` here?

Reply:
It seems like `tests.helper.get_all_tripy_interfaces` doesn't work anymore

Reply:
Could you elaborate? We use it for our docstring testing, so it should still be working.

Reply:
I keep getting the following error when calling it:

`test_dtype_constraints.py:29: in <module>
    print(get_all_tripy_interfaces())
../../tripy/common/datatype.py:48: in __repr__
    return cls.name
E   AttributeError: type object 'BaseDtype' has no attribute 'name`

Reply:
Oh this is an issue do to me trying to print the result.

Reply:
I'll fix that in https://github.com/NVIDIA/TensorRT-Incubator/pull/170